.github/workflows/test-server.yml

```diff
@@ -0,0 +1,65 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Unit test server with Postgres
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+permissions:
+  contents: read
+
+jobs:
+  # Label of the container job
+  container-job:
+    # Containers must run in Linux based operating systems
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        python-version: ["3.10"]
+
+    # Service containers to run with `container-job`
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres
+        ports:
+          - 5432:5432
+        env:
+          # Provide the password for postgres
+          POSTGRES_PASSWORD: postgres
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      # Downloads a copy of the code in your repository before running CI tests
+      - name: Check out repository code
+        uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: 'pip'
+          cache-dependency-path: 'server/requirements.txt'
+
+      - name: Install dependencies
+        run: pip install -r server/requirements.txt
+
+      - name: Setup Postgres database
+        run: |
+          cp server/tests/.svarog.ini.github server/svarog.ini
+          mkdir -p ~/.config/svarog-gut
+      - name: Perform unit tests
+        run: |
+          cd server
+          python -m unittest discover
```

Comment:
Any reason why this job is using `unittest` rather than `pytest` to run tests? I like pytest to run with `-s -v` (don't capture std output and print list of tests being run). This makes it easier to spot problems.

I'm not pushing for this, just being curious.

Reply:
We discussed this. `unittest` is marginally faster and we don't use any `pytest` features. Also, workflow logs don't preserve or show colours...

Anyway, resolving.